### PR TITLE
Change homepage url in settings

### PR DESCRIPTION
**Pull request by pazuzu156**
Changed the homepage text and link on the settings dialog to link to the Open-Shell home page.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,10 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+# ignore vscode stuff
```
Comment (ge0rdi) [inline .gitignore]: This is unrelated to URL change. Please, put it to separate commit.

File: Src/Lib/Lib.rc [hunk 2]
```diff
@@ -27,18 +27,18 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 // TEXTINCLUDE
 //
 
-1 TEXTINCLUDE
```
Comment (ge0rdi) [inline Src/Lib/Lib.rc]: Please remove all these unnecessary white-space changes.

Comment (pazuzu156) [reply Src/Lib/Lib.rc]: The editor itself did that, I didn't. I'll do what I can though.

File: .gitignore (hunk 1 above)
Comment (pazuzu156) [reply .gitignore]: Moved to another commit

File: Src/Lib/Lib.rc (hunk 2 above)
Comment (pazuzu156) [reply Src/Lib/Lib.rc]: Whitespace changes applied. Back to original with text changes for commit

Comment (ge0rdi) [reply Src/Lib/Lib.rc]: Thanks.
Now please squash those two commits with `Change homepage url in settings` message into one commit. So that it will contain just changes you meant to do.

Comment (pazuzu156) [reply Src/Lib/Lib.rc]: it's done

Comment (ge0rdi) [reply Src/Lib/Lib.rc]: You should end up with 2 commits - `Change homepage url in settings`, `ignore vscode generated files`.
You can easily rearrange commits using [interactive rebase](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history) and then just force push (`git push -f`).

Comment (pazuzu156) [reply Src/Lib/Lib.rc]: Done

**Comment by blackcrack** [comment]
:+1:  Well Done !
keep it up :1st_place_medal:
